discover/reference/articles/05-forms-field-types.markdown

```diff
@@ -1,50 +1,50 @@
-# Form Field Types
+# Form Field Types [](id=form-field-types)
 
-The essence of a form is the form field. Out of the box, Liferay Forms gives you
-many useful and highly configurable field types.
+A form without fields is no form at all. To meet your form-building needs,
+Liferay Forms gives you many useful and highly configurable field types.
 
-![Figure x: Out of the box form field types.](../images/forms-field-types.png)
+![Figure 1: Out of the box form field types.](../images/forms-field-types.png)
 
 Form Text
 : Users do not enter data into Form Text fields. The form creator enters text
 that form users see displayed on the form. This is useful for longer
 instructions.
 
-![Figure x: A form text field.](../images/forms-form-text.png)
+![Figure 2: A form text field.](../images/forms-form-text.png)
 
 Text Field
 : Users enter text into these fields. For example, a Full Name field will often
 be a text field. By default, a text field keeps all input on a single line of
 text. To accommodate longer responses, choose the multiline setting when
 configuring the text field.
 
-![Figure x: A multiline text form.](../images/forms-multiline.png)
+![Figure 3: A multiline text form.](../images/forms-multiline.png)
 
 Select List
```

Comment:
Hey @rbohl,

In the pop up to choose the Field Type, this field is named as **Select from List**, so, in order to keep consistency, can we name it here as **Select from List** too?